configuration/salt.md

```diff
@@ -104,132 +114,137 @@ You also can [order the execution][salt-doc-states-order] of your states:
-    qubesctl --all state.highstate
+    $ qubesctl --all state.highstate
 
-(More information on the command is further down.)
+(More information on the `qubesctl` command further down.)
 
-### Templating files
+### Template Files
 
-You will sometimes find your self writing repetitive states. To solve this,
-there is the ability to template files or states.
-This can be done with [jinja][jinja].
-Jinja is similar to python and behaves in many cases similar, but there 
-sometimes are differences (e.g. If you set some variable inside a loop,
-the variable outside will not get changed. Unless you use a do statement).
-So you should take a look at the [jinja api documentation][jinja-tmp].
-How you can use jinja to directly call salt functions and get data about
-your system is documented in the [salt documentation][jinja-call-salt-functions].
+You will sometimes find yourself writing repetitive states.
+To solve this, there is the ability to template files or states.
+This is most commonly done with [Jinja][jinja].
+Jinja is similar to Python and in many cases behaves in a similar fashion, but 
+there are sometimes differences when, for example, you set some variable inside
+a loop: the variable outside will not get changed.
+Instead, to get this behavior, you would use a `do` statement.
+So you should take a look at the [Jinja API documentation][jinja-tmp].
+Documentation about using Jinja to directly call Salt functions and get data 
+about your system can be found in the official 
+[Salt documentation][jinja-call-salt-functions].
 
-## Salt configuration, Qubes OS layout
+## Salt Configuration, QubesOS layout
 
-All salt configuration in `/srv/` directory, as usual. The main directory is
-`/srv/salt/` where all state files reside. States are contained in `*.sls`
-files. However the states that are part of standard Qubes distribution are
-mostly templates and the configuration is done in pillars from formulas.
+All Salt configuration files are in the `/srv/` directory, as usual.
+The main directory is `/srv/salt/` where all state files reside.
+States are contained in `*.sls` files.
+However, the states that are part of the standard Qubes distribution are mostly 
+templates and the configuration is done in pillars from formulas.
 
-The formulas are in `/srv/formulas`, including stock formula for domains in
+The formulas are in `/srv/formulas`, including stock formulas for domains in
 `/srv/formulas/dom0/virtual-machines-formula/qvm`, which are used by firstboot.
 
-Because we use some code that is not found in older versions of salt, there is
-a tool called `qubesctl` that should be run instead of `salt-call --local`. It
-accepts all arguments of the vanilla tool.
-
+Because we use some code that is not found in older versions of Salt, there is
+a tool called `qubesctl` that should be run instead of `salt-call --local`.
+It accepts all the same arguments of the vanilla tool.
 
-## Configuring system inside of VMs
+## Configuring Systems from Dom0
```

Comment:
This probably is true.  When reading the documentation, I was confused between this section and the heading "Example of vm system configuration" on line 265 (which I changed to "Example of Configuring Systems from Dom0" on line 290).

To me it seems like both of these sections describe configuring VMs from Dom0.  Are they in fact two completely different sections entirely and I just misunderstood?